apps/admin/src/app/dashboard/reports/[id]/page.tsx

```diff
@@ -39,7 +39,7 @@ const getComponentByType = (type: ReportWithInfo['type']) => {
     USER: UserReport,
   }[type];
 };
-export default async function ({ params: { id } }: Props) {
+export default async function ReportPage({ params: { id } }: Props) {
```

Comment:
[`react/display-name`](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/display-name.md): without a nice name for the component, debugging messages and visualizers are less informative.